main.go

```diff
@@ -84,7 +88,7 @@ func main() {
 	}
 
 	ctrl.SetLogger(zap.New(func(o *zap.Options) {
-		o.Development = true
+		o.Development = development
```

Comment:
The only concern I have here is we can either activate json logging with Info level or console logging with debugging but nothing between. Once we can update to a new kube-builder version this will be easier: https://github.com/kubernetes-sigs/controller-runtime/blob/v0.8.1/pkg/log/zap/zap.go#L279

Reply:
With the change from above we can use to pass the log level (`uzap` refers to `uzap "go.uber.org/zap"`):

```go
	logLevel := uzap.NewAtomicLevelAt(*level)
	ctrl.SetLogger(
		zap.New(
			zap.UseDevMode(development),
			zap.WriteTo(logWriter),
			zap.Level(&logLevel),
			),
		)
```

Reply:
You still need to add thee log level like in the example above. Also you must add `uzap "go.uber.org/zap"` to the imports. And you should run `make lint all` to and fix the lint issues golang complain.